apps/src/templates/studioHomepages/TeacherSections.jsx

```diff
@@ -22,81 +22,82 @@ import {
 import CoteacherInviteNotification from './CoteacherInviteNotification';
 import SetUpSections from './SetUpSections';
 
-class TeacherSections extends Component {
-  static propTypes = {
-    //Redux provided
-    asyncLoadSectionData: PropTypes.func.isRequired,
-    asyncLoadCoteacherInvite: PropTypes.func.isRequired,
-    coteacherInvite: PropTypes.object,
-    coteacherInviteForPl: PropTypes.object,
-    studentSectionIds: PropTypes.array,
-    plSectionIds: PropTypes.array,
-    hiddenPlSectionIds: PropTypes.arrayOf(PropTypes.number).isRequired,
-    hiddenStudentSectionIds: PropTypes.arrayOf(PropTypes.number).isRequired,
-    sectionsAreLoaded: PropTypes.bool,
-  };
-
-  componentDidMount() {
-    this.props.asyncLoadSectionData();
-    this.props.asyncLoadCoteacherInvite();
-  }
-
-  shouldRenderSections() {
-    return (
-      this.props.studentSectionIds?.length > 0 || !!this.props.coteacherInvite
-    );
-  }
+function TeacherSections({
+  asyncLoadSectionData,
+  asyncLoadCoteacherInvite,
+  coteacherInvite,
+  coteacherInviteForPl,
+  studentSectionIds,
+  plSectionIds,
+  hiddenPlSectionIds,
```

Comment:
I don't see this prop being used anywhere in this component. is it some legacy prop that we could get rid of, or is it missing from `shouldRenderPlSections`?

Reply:
Not sure about this - I'll ask in the teacher tools channel

Reply:
https://codedotorg.slack.com/archives/C045UAX4WKH/p1714656205422699